### DISCUSSION TITLE
no number generated for total body clearance (or AUC_inf)

### DISCUSSION THREAD

**Discussion by Fate-afshar** [Open-Systems-Pharmacology/Forum#1297] [Questions & Problems]
greetings 
I hope everyone is having a great day.
my issue is that my simulation won't generate a value for total body clearance in pk-analysis, a value I need.
is there an input value I haven't put in or  ...?
thanks 
![Screenshot (159)](https://user-images.githubusercontent.com/108145488/188812430-1e26fcb2-0ce8-42e6-975e-f9f385ea581c.png)

**Comment by Fate-afshar** [comment]
I tried increasing the  pts/h at the end of the dosing interval with no luck. what else can I do to get the values I want ?

**Comment by tobiasK2001** [comment]
Hi,
what is your dosing scheme? po? single or multiple dose?
If it is multiple dose I can probably reproduce you issue: If I dose once daily for three days and set Simulation time to 4 days total body clearance is not calculated:
![image](https://user-images.githubusercontent.com/25585248/189083139-d00037bd-1f34-4883-a673-851378776f7f.png)
But if I reduce simulation time to two days, PK-Sim calculates it: 
![image](https://user-images.githubusercontent.com/25585248/189083427-ba90aa80-77a8-443d-b056-abd95e1a1cc0.png)
I am not sure if this behaviour makes sense. Probably it is a bug.
Hope this might help you. Best Tobias

**Reply by Fate-afshar** [reply]
Thanks for the reply . It is a IV dose of vancomycin twice a day (multiple dose). Is there a way to fix this bug ? A way around it?  Thx again

**Comment by msevestre** [comment]
The behaviour that @tobiasK2001 is observing does not make sense. 

AUC_Inf and therefore CL are only calculated for **Single** administration. While we may change this behaviour in the future (/cc @PavelBal  and @Yuri05 ), this is the current behaviour. 

@Fate-afshar CL is nothing more than TotalDrugMassPerBodyWeight/AucInf. You can calculate it yourself for instance by using AUC_inf_TD1 (considering only the first dose)

Reducing the application to only use TWO administrations should still consider the application to be multiple dosing and not show AUC_inf. This is where there is potentially a bug.  @tobiasK2001 Can you share that project with us so that we can check the behaviour?

**Comment by tobiasK2001** [comment]
Yes indeed. Sane behaviour for iv multiple dose as for po multiple dose.
Please find the project attached
[sent to GitHub.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/9528204/sent.to.GitHub.zip)

**Comment by Fate-afshar** [comment]
Thank you for your solution!

**Reply by msevestre** [reply]
@tobiasK2001 I created an issue for this. It will be fixed. Interestingly, it is only when you set 2 days exactly. 47 hours does not do it

**Reply by tobiasK2001** [reply]
Thanks for that :-), and yes that is indeed strange ;-)

**Reply by msevestre** [reply]
well I found the issue. Easy fix
